### ISSUE TITLE
Errors when using the Matlab Wrapper Example on Mac OSX

### ISSUE THREAD

**Issue by etjens** [usnistgov/REFPROP-wrappers#69]
I struggled to run the example code in Matlab on my Mac. I wanted to document here some of the challenges/sites I used as I found it confusing trying to navigate the various similar sites. After many days spent working through python issues and c-make installs (it appears that version 2 of python will not work for the cmake build but I didn't see that explicitly listed anywhere and so I lost a lot of time), I was able to build the REFPROP shared library (using the steps in https://github.com/usnistgov/REFPROP-cmake), and run the python example file without issue (using the steps in https://nbviewer.jupyter.org/github/usnistgov/REFPROP-wrappers/blob/master/wrappers/python/notebooks/Tutorial.ipynb). 

I then couldn't get the Matlab version to run. To fix my issues, I open Matlab (R2015b) on my Mac and do the following: 
1. Check that python version 3.7 is being used via ```pyversion```
2. ```[v,e] = pyversion; system([e,' -m pip install --user -U ctREFPROP'])```
3. ```setenv('RPPREFIX','/Users/ejens/Documents/REFPROP-cmake/build/')```
4. ```import py.ctREFPROP.ctREFPROP.REFPROPFunctionLibrary```
5. ```RP = REFPROPFunctionLibrary('/Users/ejens/Documents/REFPROP-cmake/build/');```
6. ```RP.SETPATHdll('/Users/ejens/Documents/REFPROP-cmake/build/')```
7. Then when I check the RefProp version I get the following: 
```
RP.RPVersion()

ans = 

  Python str with no properties.

    ******
```
I was expecting the string output but thought I would also get the 10. 
8. Then I try to run the example code and get the following: 
```
r = RP.REFPROPdll('Water','PQ','T',int8(0),int8(0),int8(0),101.325,0.0,{1.0})

r = 

  Python REFPROPdlloutput with properties:

    Output
    hUnits
    herr
    iUCode
    ierr
    q
    x
    x3
    y
    z

    REFPROPdlloutput(z=array('d', [1.0]), Output=array('d', [373.1242958476996, ...
```

**Comment by ianhbell** [comment]
Actually, you are doing everything perfectly in MATLAB!  The ``RPVersion`` function is broken in the 10.0 release of REFPROP, and this is a bug we will fix in the bugfix for 10.0.1; it always returns **********


And in your matlab code, you just need to access the Output tag, so something like:


``` matlab

Output = double(r.Output)

```


to get the output(s) as a double array, and then the first entry (``Output(1)``) is the number you want (can't you see it in the output you copy-pasted 😊)

**Comment by EricLemmon** [comment]
The RPVersion routine is now working, if you would like an updated DLL, please contact us directly (eric.lemmon@nist.gov)
